dev/com.ibm.ws.microprofile.config.1.4/test/src/com/ibm/ws/microprofile/config14/test/VariableResolutionTest.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config14.test;
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigAccessor;
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+import org.junit.Test;
+
+/**
+ *
+ */
+public class VariableResolutionTest extends AbstractConfigTest {
+
+    @Test
+    public void variableResolutionTest() {
+        Config config = ConfigProviderResolver.instance().getConfig();
+        ConfigAccessor<String> accessor = config.access("layer1");
```

Comment:
These tests would be much clearer if we could get the actual variable values into the same class as the test code.

Reply:
you mean "manually" reading the properties file in?